octopusListener.js

```diff
@@ -63,7 +63,7 @@ instance.prototype.config_fields = function () {
 			id: 'info',
 			width: 12,
 			label: 'Information',
-			value: 'This module is for the Octopus Listener from <a href="http://noismada.com" target="_new">noismada.com</a>.'
+			value: 'This module is for the Octopus Listener from <a href="http://noismada.com" target="_new">noismada.com</a>. When using a custom command use the following syntax: GKS<TYPE>...*...<ENDOFTRANS> Where the first three points are modifier (use None if no modifier) and second three points are for the key'
```

Comment:
But custom command is for more than just GKS<TYPE> right?

Reply:
Add the moment not, there is no documentation available. Only thing I found are the commands send from within Octopus. So you can test an command for new stuff and then request an extra option.

Reply:
I mean you can send everything, but other than GKS<TYPE> is guessing